src/Elastic.OpenTelemetry/ElasticOpenTelemetryBuilder.cs

```diff
@@ -48,6 +49,23 @@ public class ElasticOpenTelemetryBuilder : IOpenTelemetryBuilder
 	/// <inheritdoc cref="IOpenTelemetryBuilder.Services"/>
 	public IServiceCollection Services { get; }
 
+	/// <summary>
+	/// Shared bootstrap routine for the Elastic OpenTelemetry Distribution.
+	/// Used to ensure auto instrumentation and manual instrumentation bootstrap the same way.
+	/// </summary>
+	public static (EventListener, ILogger) Bootstrap(ElasticOpenTelemetryBuilderOptions options)
```

Comment:
This method doesn't honour the same config for opting out of some of our defaults as in the original ctor?

Reply:
As discussed the auto instrumentation has their own mechanism to opt out of signals. 

Looking to normalise the config for this when working on packaging.